modules/pricer/src/test/java/com/opengamma/strata/pricer/capfloor/IborCapletFloorletSabrRateVolatilityDataSet.java

```diff
@@ -0,0 +1,166 @@
+  private static final DoubleArray EUR_FWD_RATE = DoubleArray.of(0.0150, 0.0125, 0.0150, 0.0175, 0.0175, 0.0190, 0.0200, 0.0210);
+  private static final CurveName EUR_FWD_NAME = CurveName.of("EUR-FWD");
+  private static final CurveMetadata EUR_FWD_META = Curves.zeroRates(EUR_FWD_NAME, ACT_ACT_ISDA);
+  private static final InterpolatedNodalCurve EUR_FWD_CURVE =
+      InterpolatedNodalCurve.of(EUR_FWD_META, EUR_FWD_TIME, EUR_FWD_RATE, LINEAR);
+
+  private static final DoubleArray ALPHA_TIME = DoubleArray.of(0.0, 0.5, 1.0, 2.0, 5.0, 10.0);
+  private static final DoubleArray BETA_TIME = DoubleArray.of(0.0, 0.5, 1.0, 2.0, 5.0, 10.0, 100.0);
+  private static final DoubleArray RHO_TIME = DoubleArray.of(0.0, 0.5, 1.0, 2.0, 5.0, 10.0, 100.0);
+  private static final DoubleArray NU_TIME = DoubleArray.of(0.0, 0.5, 1.0, 2.0, 5.0, 10.0, 100.0);
+  static final CurveMetadata META_ALPHA = Curves.sabrParameterByExpiry(
+      "Test-SABR-Alpha", ACT_ACT_ISDA, ValueType.SABR_ALPHA).withParameterMetadata(createParameterMetadata(ALPHA_TIME));
+  static final CurveMetadata META_BETA = Curves.sabrParameterByExpiry(
+      "Test-SABR-Beta", ACT_ACT_ISDA, ValueType.SABR_BETA).withParameterMetadata(createParameterMetadata(BETA_TIME));
+  static final CurveMetadata META_RHO = Curves.sabrParameterByExpiry(
+      "Test-SABR-Rho", ACT_ACT_ISDA, ValueType.SABR_RHO).withParameterMetadata(createParameterMetadata(RHO_TIME));
+  static final CurveMetadata META_NU = Curves.sabrParameterByExpiry(
+      "Test-SABR-Nu", ACT_ACT_ISDA, ValueType.SABR_NU).withParameterMetadata(createParameterMetadata(NU_TIME));
+
+  private static final DoubleArray ALPHA_VALUE_FLAT = DoubleArray.of(0.05, 0.05, 0.05, 0.05, 0.05, 0.05);
+  private static final DoubleArray BETA_VALUE_FLAT = DoubleArray.of(0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5);
+  private static final DoubleArray RHO_VALUE_FLAT = DoubleArray.of(-0.25, -0.25, -0.25, -0.25, -0.25, -0.25, -0.25);
+  private static final DoubleArray NU_VALUE_FLAT = DoubleArray.of(0.5, 0.5, 0.5, 0.5, 0.5, 0.5, 0.5);
+  private static final InterpolatedNodalCurve CURVE_ALPHA_FLAT = InterpolatedNodalCurve.of(
+      META_ALPHA, ALPHA_TIME, ALPHA_VALUE_FLAT, LINEAR);
+  private static final InterpolatedNodalCurve CURVE_BETA_FLAT = InterpolatedNodalCurve.of(
+      META_BETA, BETA_TIME, BETA_VALUE_FLAT, LINEAR);
+  private static final InterpolatedNodalCurve CURVE_RHO_FLAT = InterpolatedNodalCurve.of(
+      META_RHO, RHO_TIME, RHO_VALUE_FLAT, LINEAR);
+  private static final InterpolatedNodalCurve CURVE_NU_FLAT = InterpolatedNodalCurve.of(
+      META_NU, NU_TIME, NU_VALUE_FLAT, LINEAR);
+  static final SabrParameters SABR_PARAM_FLAT = SabrParameters.of(
+      CURVE_ALPHA_FLAT, CURVE_BETA_FLAT, CURVE_RHO_FLAT, CURVE_NU_FLAT, SabrVolatilityFormula.hagan());
+
+  private static final DoubleArray ALPHA_VALUE = DoubleArray.of(0.05, 0.05, 0.05, 0.05, 0.05, 0.05);
```

Comment:
The values of alpha, beta, etc. should not be flat in this case to test the actual interpolation is done on correct dates.